assets/js/main.js

```diff
@@ -22,7 +22,7 @@ const cleanupNotifiedIds = () => {
     .concat([worldState.cetusCycle.id])
     .concat([worldState.voidTrader.id])
     .concat(worldState.persistentEnemies.map(enemy => enemy.pid));
-  const notifiedIds = JSON.parse(localStorage.getItem('notifiedIds') || '[]');
+  const notifiedIds = JSON.parse(localStorage.notifiedIds || '[]');
```

Comment:
Is there a particular reason we're switching off of the built-in function calls?

Reply:
In case of changes to the `Storage` object, I'd prefer to stick with the function calls, especially with `localStorage.mode`.